.changeset/pretty-eels-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hono/conform-validator': patch
```

Comment:
There are differences in which version to start with regarding version numbers. Some start `0.0.1`, and some start `1.0.0`.

I think this could go either way and it's okay to release it as a major version. If you are okay with it, can you write `major` instead of `patch`?

Reply:
OK. I fixed it to update as `major`.